docs/static/releasenotes.asciidoc

```diff
@@ -11,7 +15,212 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-2-0]]
+=== Logstash 6.2.0 Release Notes
+
+* Added support to protect sensitive settings and configuration in a https://www.elastic.co/guide/en/logstash/current/keystore.html[keystore]
+* https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines] is now the default configuration (applicable for RPM/DEB/Docker installations)
```

Comment:
Can we embellish what this means for users? Will there be any breakage on upgrade? Do they need to take any new action WRT pipelines.yml?

Reply:
Put another way, is this a breaking change at all for these users.

Reply:
From 6.1 defaults to 6.2 defaults it is passive. ... If users deviated from the defaults and/or use some custom installation this _could_ , but not necessarily will break them. Most likely it should be passive. Here is the diff in question: https://github.com/elastic/logstash/commit/e12090667359044da4c746740dd9bebfb54dee87